hazelcast-hibernate/hazelcast-hibernate3/src/main/java/com/hazelcast/hibernate/serialization/ExpiryMarker.java

```diff
@@ -33,7 +33,7 @@
  *     <li>It will always return a null value, resulting in a cache miss</li>
  *     <li>It is only replaceable when it is completely expired</li>
  *     <li>It can be marked by multiple transactions at the same time and will not expire until all transactions complete</li>
- *     <li>It should not be expired unless {@link #matches(SoftLock)} is true</li>
+ *     <li>It should not be expired unless  is true</li>
  * </ul>
  */
 public class ExpiryMarker extends Expirable implements SoftLock {
```

Comment:
@mesutcelik  Yes, did not notice that. `SoftLock` is only used by Hibernate as a marker interface. `ExpiryMarker ` does not implement `SoftLock` now.  I have added new `MarkerWrapper` which only implements `SoftLock` and acts as a DTO between hibernate and IMap.  

`SoftLock` interface is used by `RegionAccessStrategy` interface of hibernate. No class used inside EntryProcessor need to implement it.
 
https://docs.jboss.org/hibernate/orm/4.2/javadocs/org/hibernate/cache/spi/access/RegionAccessStrategy.html